mne/viz/epochs.py

```diff
@@ -442,6 +443,251 @@ def plot_epochs(epochs, epoch_idx=None, picks=None, scalings=None,
+    duration = len(epochs.times) * n_epochs
+    epoch_data = epochs.get_data()
+    if picks is None:
+        if any('ICA' in k for k in epochs.ch_names):
+            picks = pick_types(epochs.info, misc=True, ref_meg=False,
+                               exclude=[])
+        else:
+            picks = pick_types(epochs.info, meg=True, eeg=True, eog=True,
+                               ref_meg=False, exclude=[])
+    if len(picks) < 1:
+        raise RuntimeError('No appropriate channels found. Please'
+                           ' check your picks')
+
+    picks = np.sort(picks)  # to make scaling work properly
+    n_channels = np.min([n_channels, len(picks)])
+    times = epochs.times * 1e3
+
+    # Reorganize channels
+    inds = list()
+    types = list()
+    for t in ['grad', 'mag']:
+        idxs = pick_types(epochs.info, meg=t, ref_meg=False, exclude=[])
+        if len(idxs) < 1:
+            continue
+        mask = np.in1d(idxs, picks, assume_unique=True)
+        inds.append(idxs[mask])
+        types += [t] * len(inds[-1])
+    pick_kwargs = dict(meg=False, ref_meg=False, exclude=[])
+    for t in ['eeg', 'eog', 'ecg', 'emg', 'ref_meg', 'stim', 'resp',
+              'misc', 'chpi', 'syst', 'ias', 'exci']:
+        pick_kwargs[t] = True
+        idxs = pick_types(epochs.info, **pick_kwargs)
+        if len(idxs) < 1:
+            continue
+        mask = np.in1d(idxs, picks, assume_unique=True)
+        inds.append(idxs[mask])
+        types += [t] * len(inds[-1])
+        pick_kwargs[t] = False
+    inds = np.concatenate(inds).astype(int)
+    if not len(inds) == len(picks):
+        raise RuntimeError('Some channels not classified. Please'
+                           ' check your picks')
+
+    # preallocation needed for min / max scaling
+    data = np.zeros((len(epochs.events), epochs.info['nchan'], len(times)))
+
+    for ii, epoch in enumerate(epoch_data):
+        pick = 0
+        for ind in inds:
+            data[ii, pick] = epoch[ind] / scalings[types[pick]]
+            pick += 1
+
+    # set up plotting
+    size = get_config('MNE_BROWSE_RAW_SIZE')
+    if size is not None:
+        size = size.split(',')
+        size = tuple(float(s) for s in size)
+    if fig_title is None:
+        fig_title = epochs.name
+        if len(fig_title) == 0:  # empty list or absent key
+            fig_title = ''
+    fig = figure_nobar(figsize=size)
+    ax = plt.subplot2grid((10, 15), (0, 0), colspan=14, rowspan=9)
+    ax.set_title(fig_title, fontsize=12)
+    ax.axis([0, duration, 0, 200])
+    ax_hscroll = plt.subplot2grid((10, 15), (9, 0), colspan=14)
+    ax_hscroll.get_yaxis().set_visible(False)
+    ax_hscroll.set_xlabel('Epochs')
+    ax_vscroll = plt.subplot2grid((10, 15), (0, 14), rowspan=9)
+    ax_vscroll.set_axis_off()
+
+    ax_vscroll.add_patch(mpl.patches.Rectangle((0, 0), 1, len(picks),
+                                               facecolor='w', zorder=2))
+    # populate vertical and horizontal scrollbars
```

Comment:
And actually they are already a bit different. I think I'll change later if needed. And I'm not even convinced it's worth the effort at this point just to save a couple of lines of code.